File: Java/Exp_03 - Lowercase to Uppercase.java
/** Get the input string
* Convert lower case to upper case 
* Print the output
*/

import java.util.Scanner;

public class Main
{
    public static void main(String[] input)
    {
        char ch;
        int temp;
        Scanner scan = new Scanner(System.in);

        ch = scan.next().charAt(0);
		
        temp = (int) ch;
        temp = temp - 32;
        ch = (char) temp;
		
        System.out.print(ch);
    }
}

/** (1)
* Input: sathyabama
* Output: SATHYABAMA

* (2)
* Input: computer
* Output: COMPUTER

* (3)
* Input: University
* Output: UNIVERSITY
*/

File: C/Exp_03 - Lowercase to Uppercase.c
//  C Program to convert lowercase string to uppercase string

// Get the string

// convert from lower to upper using for loop

#include<stdio.h>
#include<string.h>
int main(){
    char str[25];
    int i;
    scanf("%s",str);
    for(i=0;i<=strlen(str);i++){
        if(str[i]>=97&&str[i]<=122)
          str[i]=str[i]-32;
    }
    printf("%s",str);
    return 0;
}

// Input: sathyabama
// Output: SATHYABAMA
